src/BinaryParsers/ElfBinary/ElfUtility.cs

```diff
@@ -94,5 +95,27 @@ internal static string[] NullTermAsciiToStrings(byte[] data)
             }
             return strings.ToArray();
         }
+
+        /// <summary>
+        /// Convert object list to csv.
+        /// </summary>
+        /// <param name="objectlist">object list.</param>
+        /// <param name="separator">separator, can be tab for tsv format.</param>
+        /// <param name="header">if to print the header.</param>
+        /// <returns>csv format fo the object list.</returns>
+        public static IEnumerable<string> ToCsv<T>(IEnumerable<T> objectlist, string separator = ",", bool header = true)
```

Comment:
>ToCsv [](http://example.com/codeflow?start=42&length=5)

do u need this?

Reply:
no, this was the method I used to generate that csv file in email, thought if we want to keep it for future use. I will remove it.

Reply:
removed